recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -51,16 +56,6 @@ def configure(self):
             del self.options.fPIC
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
-        if self.settings.os == "Macos":
-            # when running gdk-pixbuf-query-loaders
-            # dyld: malformed mach-o: load commands size (97560) > 32768
-            raise ConanInvalidConfiguration("This package does not support Macos currently")
-
-    def build_requirements(self):
-        self.build_requires("meson/0.61.2")
-        self.build_requires("pkgconf/1.7.4")
-        if self.options.with_introspection:
-            self.build_requires("gobject-introspection/1.70.0")
 
     def requirements(self):
         self.requires("glib/2.70.4")
```

Comment:
```suggestion
        self.requires("glib/2.72.0")
```

Reply:
nop, see previous commits

Reply:
Please put a comment on this line, or someone(including me) will inevitably try to bump it later.

Reply:
Also, what is the interaction between glib and the LLVM issue posted above?

Reply:
what is this LLVM issue?

Reply:
The one in https://github.com/conan-io/conan-center-index/pull/10154#issuecomment-1094224794

Reply:
besides switching to `compiler-rt`, the workaround could be to define function `__muloti4`, e.g. like here:
https://opensource.apple.com/source/clang/clang-600.0.54/src/projects/compiler-rt/lib/muloti4.c.auto.html
the bug seems to be old enough

Reply:
https://github.com/conan-io/conan-center-index/pull/10154/commits/d055044b85f68aa6a1b1076aa565b2e03d8a3c5a